app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -132,7 +132,14 @@ class HomePresenter(
                 view?.showRefreshing(true)
                 userInteractor.loadAllTokensDataIfEmpty()
                 val solTokens = userInteractor.loadUserTokensAndUpdateLocal(tokenKeyProvider.publicKey.toPublicKey())
-                async { userInteractor.loadUserRates(solTokens) }
+                async {
+                    try {
+                        userInteractor.loadUserRates(solTokens)
+                    } catch (t: Throwable) {
+                        Timber.e(t, "Error on loading user rates")
```

Comment:
no need to Timber.e this stuff, only Timber.i

Reply:
try catch will not handle error without using `await()`